modules/custom/az_paragraphs/src/Plugin/Field/FieldFormatter/AZBackgroundMediaFormatter.php

```diff
@@ -0,0 +1,598 @@
+<?php
+
+namespace Drupal\az_paragraphs\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceFormatterBase;
+use Drupal\media\MediaInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Url;
+use Drupal\responsive_image\Entity\ResponsiveImageStyle;
```

Comment:
This isn't needed if you make my suggested change in `getResponsiveImageStyles()`.
```suggestion
```